File: EserciziAcademy/academy/esercizi/Persona.java
package academy.esercizi;

public class Persona implements Comparable<Persona>{
    String nome;
    int eta;
    public Persona(String nome, int eta) {
        super();
        this.nome = nome;
        this.eta = eta;
    }
    @Override
    public int compareTo(Persona o) {
        return this.nome.compareTo(o.nome);
    }
}

File: corso-angular/workspace/angular/Giorno9/spiegazionehttp-observable/src/app/model/persona.ts

export class Persona {
    private _id: number = 0;
    nome: string = '';
    cognome?: string;
  
    constructor(id: number, nome: string, cognome?: string) {
      this._id = id;
      this.nome = nome;
      this.cognome = cognome;
    }
  
    get id() {
      return this._id;
    }
  
    set id(id: number) {
      this._id = id;
    }
  
  }
  